### PR TITLE
Autologon credentials update

### PR DESCRIPTION
**Pull request by roshni-vachhani**
Updating auto logon credentials guidance to advise against deleting credentials.

### CODE REVIEW

**Review by roshni-vachhani — COMMENTED** [review]
Updating the auto logon credentials guidance to advise against deleting credentials.

File: microsoft-365/test-base/contentguideline.md [hunk 1]
```diff
@@ -76,19 +76,21 @@ Other times, your zip file may have your files or content right underneath it (f
 - Error return code
 - Validate success
 - Logging to script specific log folder
+- Autologon credentials if intending to run packages that interact with the Windows UI
 
 Each script needs to run unattended (no user prompts) to successfully execute in the test pipeline.
 
-> [!NOTE]
+>
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/contentguideline.md]:
```suggestion
> [!NOTE]
```

File: microsoft-365/test-base/contentguideline.md [hunk 2]
```diff
@@ -76,19 +76,21 @@ Other times, your zip file may have your files or content right underneath it (f
...
-> [!NOTE]
+> 
 > Scripts should return "0" on successful completion and a non-zero error code if any error occurs during execution.
 
 Each script should validate that it ran successfully. For example, the install script should check for the existence of certain binaries and/or registry keys on the system after the installer binary finishes executing. This check helps to ensure with a reasonable degree of confidence that the installation was successful.
 
 Validation is necessary to properly diagnose where errors occur during a test run. For example, if the script is unable to install the application successfully versus being unable to launch it.
 
-> [!IMPORTANT]
+>
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/contentguideline.md]:
```suggestion
> [!IMPORTANT]
```

File: microsoft-365/test-base/contentguideline.md [hunk 3]
```diff
@@ -76,19 +76,21 @@ Other times, your zip file may have your files or content right underneath it (f
...
+> 
 > Scripts should return "0" on successful completion and a non-zero error code if any error occurs during execution.
 
 Each script should validate that it ran successfully. For example, the install script should check for the existence of certain binaries and/or registry keys on the system after the installer binary finishes executing. This check helps to ensure with a reasonable degree of confidence that the installation was successful.
 
 Validation is necessary to properly diagnose where errors occur during a test run. For example, if the script is unable to install the application successfully versus being unable to launch it.
 
-> [!IMPORTANT]
+> 
 > **Avoid the following:**
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/contentguideline.md]:
```suggestion
```

File: microsoft-365/test-base/contentguideline.md [hunk 4]
```diff
@@ -76,19 +76,21 @@ Other times, your zip file may have your files or content right underneath it (f
...
 > Scripts should not reboot the machine, if a reboot is necessary please specify this during the upload of your scripts.
+> Scripts that intend to interact with the Windows UI should not erase the autologon credentials. Erasing these credentials will prevent the scripts from executing correctly.
```
Comment (JohanFreelancer9) [inline microsoft-365/test-base/contentguideline.md:93]:
```suggestion
>
> Scripts that intend to interact with the Windows UI should not erase the autologon credentials. Erasing these credentials will prevent the scripts from executing correctly.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @roshni-vachhani, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@roshni-vachhani gentle reminder

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Tinacyt Please merge this PR. Thanks

**Comment by jujiang-msft** [comment]
@Tinacyt , per offline discussion with @roshni-vachhani this change will need to be merged to main.
@roshni-vachhani can you please close this once new PR is created? Thanks.

**Comment by roshni-vachhani** [comment]
Closing and opening a new one against the main branch

**Comment by yogkumgit** [comment]
@roshni-vachhani can this PR be closed.

**Pull request closed without merging**